pod-operation/hx_wheel_encodeer

```diff
@@ -0,0 +1,39 @@
+use std::thread::sleep;
```

Comment:
Issue: Is there a reason why this file doesn't have `.rs` extension on it? Also, being somewhat nitpicky here, but it should be "encoder" instead of "encodeer", unless you have a reason for intentionally spelling it this way.

Reply:
It was a typo, thanks for pointing it out!

Reply:
Regarding the `.rs` issue, you could just run `git mv pod-operation/hx_wheel_encodeer pod-operation/components/wheel_encoder.rs` (since we should organize all of our sensors into one `components/` folder and then you can commit that change.

Reply:
Thanks! Does it show up as .rs now for everyone? 
File name - resolved